### PR TITLE
test: Add unit test for seata-tm module

### PR DESCRIPTION
**Pull request by wangwei-ying**
### Ⅰ. Describe what this PR did
test: Add unit test to seata-tm module

### Ⅱ. Does this pull request fix one issue?

#194

### CODE REVIEW

**Comment by zjinlei** [comment]
Thanks

File: tm/src/test/java/io/seata/tm/TransactionManagerHolderTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.tm;
+
+import io.seata.common.exception.ShouldNeverHappenException;
+import io.seata.tm.api.transaction.RollbackRule;
```
Comment (xingfudeshi) [inline tm/src/test/java/io/seata/tm/TransactionManagerHolderTest.java]: Remove this import io.seata.tm.api.transaction.RollbackRule.

File: tm/src/test/java/io/seata/tm/api/DefaultFailureHandlerImplTest.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+/**
+ * @author wangwei
+ * @date 2019/10/24
+ */
+class DefaultFailureHandlerImplTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultFailureHandlerImplTest.class);
+
+    private static final String DEFAULT_XID = "1234567890";
+    private static final String TX_NAME = "test";
+    private static final int TIME_OUT = 30000;
```
[the remark this answers is not in the document]
Comment (keygod) [reply tm/src/test/java/io/seata/tm/api/DefaultFailureHandlerImplTest.java]: ths,all done

File: tm/src/test/java/io/seata/tm/api/DefaultFailureHandlerImplTest.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+ * @author wangwei
+ * @date 2019/10/24
+ */
+class DefaultFailureHandlerImplTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultFailureHandlerImplTest.class);
+
+    private static final String DEFAULT_XID = "1234567890";
+    private static GlobalStatus globalStatus = GlobalStatus.Begin;
+
+//    private GlobalStatus globalStatus = null;
```
Comment (xingfudeshi) [inline tm/src/test/java/io/seata/tm/api/DefaultFailureHandlerImplTest.java]: remove this line.

Comment (keygod) [reply tm/src/test/java/io/seata/tm/api/DefaultFailureHandlerImplTest.java]: done.

**Comment by slievrly** [comment]
@wangwei-ying thank you very much.

**Pull request merged**